packages/resourcehacker/PKGBUILD

```diff
@@ -1,23 +1,22 @@
 # This file is part of BlackArch Linux ( http://blackarch.org ).
 # See COPYING for license details.
 
-pkgname='resourcehacker'
+pkgname=resourcehacker
 pkgver=5.1.6
 pkgrel=1
-pkgdesc='Resource compiler and decompiler for Windows® applications'
-groups=('blackarch' 'blackarch-windows')
+pkgdesc='Resource compiler and decompiler for Windows® applications.'
+groups=('blackarch' 'blackarch-windows' 'blackarch-reversing')
 arch=('any')
 url='http://www.angusj.com/resourcehacker/'
 depends=()
+source=('http://www.angusj.com/resourcehacker/resource_hacker.zip')
+sha256sums=('de24309105f589d0c4f6d19c85d0354c0c8ffe9a2f34054ab2028b4a380c664d')
 
 pkgver() {
-	curl 'http://www.angusj.com/resourcehacker/' | grep '>Version ' | sed -r 's/.*Version ([0-9]+\.[0-9]+[^<]+).*/\1/gi'
+  curl 'http://www.angusj.com/resourcehacker/' | grep '>Version ' | sed -r 's/.*Version ([0-9]+\.[0-9]+[^<]+).*/\1/gi'
 }
 package() {
-	tmpFile='/tmp/resourcehacker.zip'
-	targetDir="${pkgdir}/usr/share/windows/resourcehacker/"
-	mkdir -p "${targetDir}"
-	curl -L --output "${tmpFile}" 'http://www.angusj.com/resourcehacker/resource_hacker.zip'
-	unzip -d "${targetDir}" "${tmpFile}"
-	rm -rf "${tmpFile}"
+  rm -rf './resource_hacker.zip'
+  mkdir -p "${pkgdir}/usr/share/windows/resourcehacker/"
+  cp -r ./* "${pkgdir}/usr/share/windows/resourcehacker/"
```

Comment:
This should be installed to `/opt` instead since it doesn't seem to have the source available. Sorry for not mentioning it in the first review.